assets/src/amp-validation/counts/index.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * External dependencies
+ */
+import { isPlainObject } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import apiFetch from '@wordpress/api-fetch';
+import domReady from '@wordpress/dom-ready';
+
+/**
+ * Internal dependencies
+ */
+import './style.css';
+
+/**
+ * Updates a menu item with its count.
+ *
+ * If the count is not a number or is `0`, the element that contains the count is instead removed (as it would be no longer relevant).
+ *
+ * @param {HTMLElement} itemEl Menu item element.
+ * @param {number} count Count to set.
+ */
+function updateMenuItem( itemEl, count ) {
+	count = Math.abs( count );
+
+	if ( isNaN( count ) || count === 0 ) {
+		itemEl.parentNode.parentNode.removeChild( itemEl.parentNode );
+	} else {
+		itemEl.classList.remove( 'loading' );
+		itemEl.textContent = count.toLocaleString();
+	}
+}
+
+/**
+ * Updates the 'Validated URLs' and 'Error Index' menu items with their respective unreviewed count.
+ *
+ * @param {Object} counts Counts for menu items.
```

Comment:
Note the `! isPlainObject( counts )` check implies that it may not be an `Object`.

Reply:
Ah right, will update this to `any`.